src/schema/schemadefinitioninspector.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import {
+	setSchemaCurrentDefinitionName
+} from './data/actions';
+
+import Pane from '../components/pane';
+import ObjectInspector from '../components/objectinspector';
+
+class SchemaDefinitionInspector extends Component {
+	render() {
+		const definition = this.props.currentSchemaDefinition;
+
+		if ( !definition ) {
+			return <Pane isEmpty="true">
+				<p>Select a schema definition to inspect</p>
```

Comment:
```suggestion
				<p>Select a schema definition to inspect.</p>
```

In the `src/schema/pane.js` file, a sentence ends with a dot.